src/features/users/usersSlice.ts

```diff
@@ -0,0 +1,47 @@
+import { createAsyncThunk, createSlice, PayloadAction } from '@reduxjs/toolkit';
+import { User } from '../../types/User';
```

Comment:
Ensure that the `User` type is correctly defined and imported from the `../../types/User` file. Verify its existence and correctness.